components/common/BlogLogo.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import Image from 'next/image';
+import Link from 'components/common/Link';
+import useTheme from 'hooks/useTheme';
+import styles from './BlogLogo.module.css';
+
+import { resizeImage, getBlurHash } from 'utils/image';
+
+export default function BlogLogo(props) {
+  const { publication } = props;
+  const [theme] = useTheme();
+  const isThemeDark = theme === 'dark';
+
+  if (!publication) {
+    return null;
+  }
+
+  return (
+    <div>
+      {publication.darkModeLogo && isThemeDark ? (
```

Comment:
@kieran6roberts Can you state how the order of these ternary expressions work ?  It's less readable at the moment. Also if these can be extracted into a separate function and just the prop values can be changed, that would be good ? 

P.S. - Want to understand the branching here.

Reply:
This was just taken from the blogs repo initially, but I have refactored this to hopefully be easier to understand here. Maybe then I could refactor it over there while I'm working on the blog navbar updates.